src/sensors.c

```diff
@@ -367,6 +367,7 @@ static int sensors_load_conf(void) {
           (feature->type != SENSORS_FEATURE_TEMP) &&
 #if SENSORS_API_VERSION >= 0x402
           (feature->type != SENSORS_FEATURE_CURR) &&
+          (feature->type != SENSORS_FEATURE_HUMIDITY) &&
```

Comment:
Unfortunately, this was not enough.
As you can see, CI check still failing:

```
CC       src/sensors_la-sensors.lo
src/sensors.c: In function 'sensors_load_conf':
src/sensors.c:370: error: 'SENSORS_FEATURE_HUMIDITY' undeclared (first use in this function)
src/sensors.c:370: error: (Each undeclared identifier is reported only once
src/sensors.c:370: error: for each function it appears in.)
src/sensors.c:390: error: 'SENSORS_SUBFEATURE_HUMIDITY_INPUT' undeclared (first use in this function)
src/sensors.c: In function 'sensors_read':
src/sensors.c:525: error: 'SENSORS_FEATURE_HUMIDITY' undeclared (first use in this function)
make[1]: *** [src/sensors_la-sensors.lo] Error 1
```

That is caused by old `lm_sensors` version used in `Centos 6`. 
There is `3.1.1` version used, which has `SENSORS_API_VERSION` set to `0x420`.
The `lm-sensors-3.3.0`, there humidity sensors support firstly added, has `SENSORS_API_VERSION 0x431`. Therefore, we need its own condition for humidity sensors. 

Please, update the code. Thanks.

Reply:
This is fixed now